### PR TITLE
Change tooltip color in dark mode

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Changed tooltip color in dark mode to "neutralLighter"(#383735) 

# Why
(https://skype.visualstudio.com/SPOOL/_workitems/edit/2658072)

# How Tested
Tooltip color in light mode should not be changed  (still verifying with Alex P)
Screen Shot 2022-04-08 at 9 47 32 AM
Screen Shot 2022-04-08 at 9 47 32 AM

tooltip color in dark mode is changed to  "neutralLighter"(#383735) 
Screen Shot 2022-04-08 at 9 47 45 AM

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

no

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/ControlButtonTooltip.tsx [hunk 1]
```diff
@@ -18,6 +23,15 @@ const hostStyles: Partial<ITooltipHostStyles> = { root: { display: 'inline-block
...
  */
 export const ControlButtonTooltip = (props: ITooltipHostProps): JSX.Element => {
+  const theme = useTheme();
+  const calloutStyle: Partial<ICalloutContentStyles> = { root: { padding: 0 }, calloutMain: { padding: '0.5rem' } };
+
+  // Place callout with no gap between it and the button.
+  const calloutProps = {
+    gapSpace: 0,
+    styles: calloutStyle,
+    backgroundColor: isDarkThemed(theme) ? theme.palette.neutralLighter : ''
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ControlButtonTooltip.tsx:33]: Does this background color need to be updated for every tooltip we have? Or just control bar buttons. (Do we mind having inconsistent tooltip backgrounds?)

Comment (carocao-msft) [reply packages/react-components/src/components/ControlButtonTooltip.tsx:33]: Hmmm good question, let me ask Alex P

Comment (JamesBurnside) [reply packages/react-components/src/components/ControlButtonTooltip.tsx:33]: Note: (we may not care about this but) if a customer is using fluent components alongside our components/composites, the existing tooltips on their site will not be consistent with ours if we update them all

Comment (carocao-msft) [reply packages/react-components/src/components/ControlButtonTooltip.tsx:33]: Updated background color for all tooltips

**Review by JamesBurnside — APPROVED** [review]

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**